### DISCUSSION TITLE
Help me!

### DISCUSSION THREAD

**Discussion by Q3544** [community/community#141954] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I have an account called qymx1024, I can not log in, because its email address was accidentally cancelled by me, how to do?

**Comment by LiteBrite82** [comment]
Hi @Q3544 ! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)
